tests/cases/services/create-service.spec.js

```diff
@@ -0,0 +1,136 @@
+var HomePage = require("../../util/HomePage");
+var Sidebar = require("../../util/Sidebar");
+var ListServicesPage = require("../../util/ListServicesPage");
+var CreateServicePage = require("../../util/CreateServicePage");
+var KongDashboard = require("../../util/KongDashboard");
+var Kong = require("../../util/KongClient");
+var request = require("../../../lib/request");
+var PropertyInput = require('../../util/PropertyInput');
+var using = require('jasmine-data-provider');
+
+var kd = new KongDashboard();
+
+describe('Service Creation testing', () => {
+    // Only run tests if the Kong version is 0.13 as Services are supported in version 0.13 only
+    if (process.env.KONG_VERSION === '0.13') {
```

Comment:
Here you'll want to check if Kong version is 0.13 **or higher**.

You can use semver for that:
```
var semver = require('semver');
if (semver.lt(process.env.KONG_VERSION, '0.13.0')) {
  // skip tests
} else {
  // run tests
}
```

Reply:
Nice suggestion.

However, I am not able to use semver to compare the versions. I am getting the following error if I use semver in tests:

```
Service Creation testing encountered a declaration exception
  Message:
    TypeError: Invalid Version: 0.12
  Stack:
    TypeError: Invalid Version: 0.12
        at new SemVer (/Users/pkumar/kong-dashboard/node_modules/semver/semver.js:305:11)
        at compare (/Users/pkumar/kong-dashboard/node_modules/semver/semver.js:578:10)
        at Function.lt (/Users/pkumar/kong-dashboard/node_modules/semver/semver.js:612:10)
        at Suite.fdescribe (/Users/pkumar/kong-dashboard/tests/cases/services/create-service.spec.js:16:16)
        at Object.<anonymous> (/Users/pkumar/kong-dashboard/tests/cases/services/create-service.spec.js:14:1)
        at Module._compile (internal/modules/cjs/loader.js:702:30)
        at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)
```

I need to spend more time figuring this issue out. Looks like by node (v10.2.1) and npm (5.6.0) versions are not liking Semver. Let me know if this is a blocker for merging this pull request. I will be working on route integration next and will revisit this issue once I write tests for routes.

Reply:
Ok no worry I'll take care of that.